### PR TITLE
Trying to fix the previous PR.  (menu.py)

### PR DESCRIPTION
**Pull request by raduiman**
Add the new menu.py to fix the issue reported by Travis with the first PR.

### CODE REVIEW

File: Automation/menu.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import subprocess
+
+file = open("banner.txt","r")
+for line in file:
+    print(line)
+file.close()
+
+def menu():       #the menu. 8 options
+    print("\n1. Update checker. ")
```
Comment (Akhliskun) [inline Automation/menu.py]: Maybe we could have this as:
Check for updates.

File: Automation/menu.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        subprocess.call(['python', '../client.py', '-w', 'osx'])
+    elif option == 7:
+        subprocess.call("nano ../client.py")
+    elif option == 8:
+        break
+    else:
+        print("Wrong option !")
+    menu()
+
+'''A basic version of the TWC menu. I'll improve it in the next shifts.'''
```
Comment (Akhliskun) [inline Automation/menu.py]: Move this at the very top of the file.

**Comment by raduiman** [comment]
To answer to Bogdan suggestion of using switch:case instead of if-elif:
-as I know Python does not have a switch statement like Java, C, C++
-I've created this method using "if-elif-else" block with the same behavior like "switch:case" statement.

Please let me know if I'm wrong.

File: Automation/menu.py (hunk 2 above)
Comment (raduiman) [reply Automation/menu.py]: Thanks for your indications. I'll implement them.

**Comment by raduiman** [comment]
To increase the efficiency of this script we have to find a way to jump at the final of the if-elif-else block after a condition is accomplished. 
e.g if the option is '1', the first case is fulfilled and then the script is going further and check all the cases. It should exit from the 'if-elif' block and ask for another option.

**Pull request merged**